libfreerdp/core/test/TestConnect.c

```diff
@@ -241,19 +276,23 @@ static int testSuccess(void)
 
 int TestConnect(int argc, char* argv[])
 {
+	int randomPort;
+
+	randomPort = 3389 + (random() % 20);
```

Comment:
@hardening Good idea to use a random port, but maybe it would be even better if we spread the port range a little more (and maybe check availability first to avoid collisions with running services)